TerminalDocs/troubleshooting.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Windows Terminal Troubleshooting
+description: Learn fixes to common obstacles in the Windows Terminal.
+author: cinnamon-msft
+ms.author: cinnamon
+ms.date: 05/19/2020
+ms.topic: troubleshooting
+ms.service: terminal
+---
+
+# Troubleshooting in Windows Terminal
+
+Here are some common errors/obstacles you may encounter when using Windows Terminal.
+
+## How do I launch WSL in the `~` directory?
+
+By default, the `startingDirectory` of a profile is `%USERPROFILE%` (`C:\Users\<YourUsername>`). This is a Windows path. However, for WSL, you might want to use the WSL home path instead. `startingDirectory` only accepts a Windows-style path, so setting it to start within the WSL distribution can be a little tricky.
+
+Fortunately, with Windows 1903, the file systems of WSL distros can easily be addressed using the `\\wsl$\` prefix. For any WSL distro whose name is `DistroName`, you can use `\\wsl$\DistroName` as a Windows path that points to the root of that distro's file system.
+
+For example, the following works as a profile to launch the "Ubuntu-18.04" distro in its home path:
+
+```json
+{
+    "name": "Ubuntu-18.04",
+    "commandline" : "wsl -d Ubuntu-18.04",
+    "startingDirectory" : "//wsl$/Ubuntu-18.04/home/<Your Ubuntu Username>",
+}
+```
+
+## How do I set my tab title?
+
+If you'd like to have the shell automatically set your tab title, [visit the tab title tutorial](./tutorials/tab-title.md). If you want to set your tab title in your settings file, use the following steps:
+
+1. Add `"suppressApplicationTitle": true` to the profile you want to suppress any title change events that get sent from the shell. Having only this setting added to your profile will set your tab title to the name of your profile.
+
+2. If you want a custom tab title that is not the name of your profile, you can add `"tabTitle": "TITLE"` where TITLE is replaced with your preferred tab title.
+
+## Command line arguments in PowerShell
+
+PowerShell uses a semicolon `;` to delimit statements. To interpret a semicolon `;` as a command delimiter for `wt` command line arguments, you need to escape semicolon characters using backticks `` ` ``. PowerShell also has the stop parsing operator (`--%`), which instructs it to stop interpreting anything after it and just pass it on verbatim. For more information on command line arguments, visit the [Command line arguments page](./command-line-arguments.md).
```

Comment:
It might be better if we just link the relevant section of "Command line arguments" and call it a day.

Reply:
Absolutely, link that page. There are more intricacies than just escaping args (unfortunately)

Reply:
The page is linked?

Reply:
I mean literally remove the paragraph, and replace it with "visit this page". It's just repeated info and it's said waaaaay better over there.

Reply:
OH okay gotcha 👍